client/lib/DomExtender.ts

```diff
@@ -25,7 +25,8 @@ import Interactor from './Interactor';
 import { getAwaitedPathAsMethodArg } from './SetupAwaitedHandler';
 import { scriptInstance } from './internal';
 import { IExtractElementFn, IExtractElementOptions, IExtractElementsFn } from '../interfaces/IExtractElementFn';
-import CollectedElements from './CollectedElements';
+import DetachedElements from './DetachedElements';
+import DetachedDOM from './DetachedDOM';
```

Comment:
Lint error here, and there's an unused function in this class too

Reply:
Fixed